app/Game/game.cpp

```diff
@@ -50,11 +76,12 @@ int Game::run()
      */
     noecho();
```

Comment:
Move this to the Main.cpp - it is necessary for all programm, not only foer game screen.

    noecho();
    nodelay();
    set_escdelay();
    curs_set();
    keypad();